avocado/utils/memory.py

```diff
@@ -320,3 +320,56 @@ def get_thp_value(feature):
         return (re.search(r"\[(\w+)\]", value)).group(1)
     else:
         return value
+
+
+class MemInfo(object):
+    """
+    Representation of /proc/meminfo
+    """
+
+    class MemInfoItem(object):
```

Comment:
I see the intention behind declaring this class here, but I believe it'd be better to declare it in the same level as `MemInfo`. Also, this class is more akin to a `Size`, or `DataSize` object.